css/style.css

```diff
@@ -0,0 +1,311 @@
+}
+.go-withUs{
+    font-size: 2rem;
+    width: 15rem;
+    padding: auto;
+    margin: auto;
+}
+.go-to-contact{
+    background-color: var(--while-color);
+    font-weight: 400;
+    border-radius: 0.5rem;
+    box-shadow: 12px 12px 2px 1px rgba(57, 57, 65, 0.2);
+    font-family: 'Work Sans', sans-serif;
+    text-transform: uppercase;
+}
+.image-text {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: var(--while-color);
+    padding: 0.2rem;
+    font-size: 14px;
+    text-align: center;
+    opacity: 0; /* Textul va fi ascuns initial */
+    transition: opacity 0.3s ease-in-out;
+}
+.image-container:hover .image-text {
+    opacity: 1; /* textul va aparea dupa :hover */
+}
+.newsletter{
+    background-color: var(--green-color);
+    border-radius: 2rem;
+    justify-content: center;
+    text-align: center;
+}
+.newsletter-form{
+    padding: 1.5rem;
+}
+.input{
+    border-radius: 0.3rem;
+}
+.submit{
+    border-radius: 0.3rem ;
+}
+.other-font{
+    font-family: 'Work Sans', sans-serif;
+    font-size: 0.9rem;
+}
+.other-font:hover{
+    color: var(--buttons-color);
+}
+.line{
+    border: 1px solid var(--line-color);
+}
+.footer {
+    padding: 1rem;
+    margin: auto;
+    display: flex;
+    justify-content: center;
+    font-size: 1rem;
+}
+.icons{
+    justify-content: center;
+    padding: 1rem;
+    font-size: 2rem;
+}
+.fa{
+    margin: 10px;
+    align-items: center;
+}
+.fa-linkedin-square{
+    color: var(--linkedin-color);
+}
+.go-to-exeditiond{
+    border: 1px solid var(--black-color);
+    margin: 0 auto;
+    padding: 1rem;
+    background: var(--buttons-color);
+    align-items: center;
+
+}
+.center{
+    text-align: center;
+    margin: auto;
+}
+
+.content{
+    border: 1px solid var(--black-color);
+    box-shadow: 10px 5px 5px var(--buttons-color);
+    font-family: 'Playfair Display', serif;
+}
+#map{
+    height: 400px;
+}
+@media (max-width: 600px) {  
+    .img-row{
+        grid-template-columns: auto;
+    }
+    .center-height {
+        justify-content: center;
+    }
+    .header{
+        flex-direction: column;
+    }
+    .media-go{
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+    }
+    .header-logo{
+        justify-content: center;
+    }
+    .row-info {
+        flex-direction: column; 
+        text-align: center;
+        margin: auto;
+    }
+    .col-4 {
+        flex: none;
+        width: 100%; 
+}
+    .col-2{
+        display: flex;
+        width: 100%;
+        align-items: center;
+        flex-direction: row;
+    }
+    .line{
+        width: 100%;
+        margin: auto;
+    }
+    .title{
+        align-items: center;
+        justify-content: center;
+    }
+    .content{
+        box-shadow: 10px 5px 5px 10px var(--buttons-color);
+    }
+}
+@media(min-width: 700px) and (max-width:1184px) {
+    .col-2{
+        flex: none;
```

Comment:
Pe pagina de contact meniul ajunge puțin deasupra conținutului la acest media query.